### PR TITLE
[jsk_perception] Add trained maskrcnn model for 73b2 kitchen

### PR DESCRIPTION
**Pull request by iory**
I add pre trained mask rcnn model for 73b2 kitchen.
Following image is result of mask rcnn output.

left0001

Using this model, we can detect 43 classes.
Following is the list of the detectable class name.
```
  - dish
  - cup
  - spoon
  - fork
  - knife
  - tray
  - kettle
  - plasticbottle
  - paperpack
  - can
  - microwave
  - refrigerator
  - chair
  - table
  - tv/monitor
  - cupnoodle
  - cereal
  - cornpotage
  - teabag
  - alfort
  - alfortwhite
  - almond
  - apc_shelf_flont
  - chipstar
  - coffee
  - consome
  - dars
  - darsmilk
  - darswhite
  - donbe
  - kinoko
  - macadamia
  - marble
  - milktea
  - mixjuice
  - norishio
  - pie
  - ritz
  - shelf_flont
  - takenoko
  - tee
  - xylitop
  - yakisoba
```

### CODE REVIEW

**Comment by k-okada** [comment]
@iory please add the list of a learned model in this PR.

**Comment by iory** [comment]
OK.
Names of the trained model are following.
```
  - dish
  - cup
  - spoon
  - fork
  - knife
  - tray
  - kettle
  - plasticbottle
  - paperpack
  - can
  - microwave
  - refrigerator
  - chair
  - table
  - tv/monitor
  - cupnoodle
  - cereal
  - cornpotage
  - teabag
  - alfort
  - alfortwhite
  - almond
  - apc_shelf_flont
  - chipstar
  - coffee
  - consome
  - dars
  - darsmilk
  - darswhite
  - donbe
  - kinoko
  - macadamia
  - marble
  - milktea
  - mixjuice
  - norishio
  - pie
  - ritz
  - shelf_flont
  - takenoko
  - tee
  - xylitop
  - yakisoba
```
Total 43

**Comment by iory** [comment]
Update description of this PR. Please check it.

**Comment by taichiH** [comment]
There are some kind of typo in pre trained model label list.
```
- tee
- xylitop
```
Sorry, this is my fault, I will fix it.

File: jsk_perception/scripts/install_trained_data.py [hunk 1]
```diff
@@ -222,6 +222,21 @@ def main():
         md5='47a507934b6bc20f0d9274825b734942',
     )
 
+    # node_scripts/mask_rcnn_instance_segmentation.py 73B2 kitchen
```
Comment (k-okada) [inline jsk_perception/scripts/install_trained_data.py]: @taichiH 
please add link to classneames.yaml to here, also create page to show trained data within 'https://drive.google.com/uc?id=1f9E2b1uJrk3eLbJmEDSSPl6TQKgKvZLW directory

**Comment by taichiH** [comment]
I fixed label name and @iory merged my commit.

**Comment by k-okada** [comment]
@taichiH please read my comment -> https://github.com/jsk-ros-pkg/jsk_recognition/pull/2423/files#r276061502

File: jsk_perception/scripts/install_trained_data.py (hunk 1 above)
Comment (taichiH) [reply jsk_perception/scripts/install_trained_data.py]: @k-okada  sorry, I don't understand meaning about "create page". Where should I create a page? Do you say add model to google drive and show recognition result on this pr thread?

**Comment by k-okada** [comment]
create README.md in google drive and show the list of times with it's
images. Otherwise, we could not know what we can recognize with this data
set.
--
◉ Kei Okada



2019年4月27日(土) 0:00 Taichi Higashide <notifications@github.com>:

**Comment by iory** [comment]
@taichiH Did you add README.md?

**Comment by iory** [comment]
@taichiH 

It seems that intstall_learning_dataset's image does not contain your annotation datasets.
https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/jsk_perception/scripts/install_learning_datasets.py#L28
https://drive.google.com/uc?id=1iBSxX7I0nFDJfYNpFEb1caSQ0nl4EVUa

So, some class could not detect well.
Could you merge the data to kitchen_dataset?

**Comment by taichiH** [comment]
@iory 
As you said, there are no my annotation datasets (convenience store items). I will merge datasets and learning. I think I can commit until this weekend.

**Comment by iory** [comment]
@taichiH updated documents and a sample launch.

**Comment by k-okada** [comment]
@YutoUchimi ちなみに73B1データセットと呼んでいるのはこれ？それとも違うもの？
Cc: @YoheiKakiuchi

**Comment by YutoUchimi** [comment]
@k-okada これです。正確には73B2キッチンデータセットとでも呼ぶべきでしょうか。

**Pull request merged**